GOVERNANCE.md

```diff
@@ -4,48 +4,47 @@
 <BR>
 
 ## Key references:
-Please familiarize yourself with our [Project Vision](./VISION.md) and [Contributing Guidelines](./CONTRIBUTING.md). Together with this Governance document, these provide useful info on what and who we are, and how we operate. 
+Please familiarize yourself with our [Project Vision](./VISION.md) and [Contributing Guidelines](./CONTRIBUTING.md). Together with this Governance document, these provide useful info on what and who we are, and how we operate.
 
 ---
 <BR>
 
-The **OpenEBS project** is managed on a daily basis by a senior governance & leadership team, that consists of the following OpenEBS ```Admins``` and ```Maintainers.``` Currently the team is...
+The **OpenEBS project** is managed on a daily basis by a senior governance & leadership team, that currently consists of the following OpenEBS ```Maintainers```:
 > |   |   |   |    |
 > | :--- | :--- | :--- | :--- |
-> | [Vishnu Attur](https://www.linkedin.com/in/vishnu-attur-5309a333/ "Bengaluru, Karnataka, India (GMT+5:30) Timezone")| :octocat: <kbd>**[@avishnu](https://github.com/avishnu "Vishnu Govind Attur")**</kbd> | ![](/images/flags/de_je/in.png) | <kbd>**Admin**</kbd>, ```Maintainer``` |
+> | [Vishnu Attur](https://www.linkedin.com/in/vishnu-attur-5309a333/ "Bengaluru, Karnataka, India (GMT+5:30) Timezone")| :octocat: <kbd>**[@avishnu](https://github.com/avishnu "Vishnu Govind Attur")**</kbd> | ![](/images/flags/de_je/in.png) | ```Maintainer``` |
```

Comment:
Ah looks like this is also repeated on the README.md
Must we keep this on both?

Reply:
Do we need to fix the admin bit in the readme as well?

Reply:
Done